### DISCUSSION TITLE
Fork and test it (publish)

### DISCUSSION THREAD

**Discussion by Mrcdkk** [statelyai/xstate#2044] [Q&A] [answered]
**Description:**

I have added a socket.io client implementation to the inspektor. I wonder how i can use this in another project now.

I forked it and published it via npm publish. But a npm install doesn´t work. The transpiled JS from the typescript code are not published.

Is there a other method to use the forked xstate, without publishing the project?

**Comment by Andarist** [comment, marked as answer]
You can pack it and install it from a local tarball.

**Reply by Mrcdkk** [reply]
Thanks. "npm pack" works fine
